test/jdk/tools/jpackage/share/AddLauncherTest.java

```diff
@@ -200,6 +202,33 @@ public void testMainLauncherIsModular(boolean mainLauncherIsModular) {
         .applyTo(cmd);
 
         cmd.executeAndAssertHelloAppImageCreated();
+
+        // verify using each additional launchers cfg file that it has the
+        // main-jar/class or module/class
+
+        CfgFile cfg = cmd.readLauncherCfgFile("ModularAppLauncher");
+        String moduleValue = cfg.getValue("Application", "app.mainmodule");
+        String mainClass = null;
+        String classpath = null;
+        TKit.assertEquals(moduleValue, JavaAppDesc.parse(
```

Comment:
The first parameter of TKit.assertEquals() function should be expected value which I assume is the value obtained with the call to JavaAppDesc.parse(...) and the second parameter is actual value that seems to be the value of moduleValue variable. So the first two parameters of TKit.assertEquals() call should be swapped.